### ISSUE TITLE
aggregations not working

### ISSUE THREAD

**Issue by guruward** [mafintosh/mongojs#210]
run aggregate in shell works.
run in mongojs on node ( with callback )
db.users.aggregate(
                [
                    { $project : {month_joined : { $month : "$createdAt" } } } ,
                    { $group : { _id : {month_joined:"$month_joined"} , number : { $sum : 1 } } },
                    { $sort : { "_id.month_joined" : 1 } }
                ]  , function(err, data) {

                console.log(err) ;
                res.status(200).send('data')
            }
         )

err
{ name: 'MongoError',
  message: 'exception: pipeline element 0 is not an object',
  errmsg: 'exception: pipeline element 0 is not an object',
  code: 15942,
  ok: 0 }

**Comment by saintedlama** [comment]
Had the same issue yesterday. Pass the pipeline objects as arguments instead of an array and everything will work like a charm:

```
db.users.aggregate(
{ $project : {month_joined : { $month : "$createdAt" } } } ,
{ $group : { _id : {month_joined:"$month_joined"} , number : { $sum : 1 } } },
{ $sort : { "_id.month_joined" : 1 } }, function(err, data) {
//...
```

**Comment by saintedlama** [comment]
See also pull request #204

**Comment by saintedlama** [comment]
Fixed by #204
